src/tree/updater_gpu_hist.cu

```diff
@@ -876,14 +926,22 @@ class GPUHistMaker : public TreeUpdater {
     omp_set_num_threads(nthread);
   }
 
```

Comment:
Do we need this old version of UpdatePredictionCache without HostDeviceVector?

Reply:
I don't think we need this, especially because the interface `TreeUpdater` now has only one version of `UpdatePredictionCache`, with HostDeviceVector.